templates/workspace_services/mlflow/terraform/web_app.tf

```diff
@@ -0,0 +1,195 @@
+resource "azurerm_storage_share" "mlflow" {
```

Comment:
Should this be a new share or use the share being mounted in to VMs as per - #1339 ? Not sure of the implications, but believe the share will need to be available to the user VMs,

Reply:
I think you are right, the artefacts seem to be persisted by the client rather than the server. Therefore, this might need to be the shared storage being used on the VMs.

Reply:
Switched to shared storage share